generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -136,6 +136,9 @@ class WithFireSimHighPerfClocking extends Config(
 
 // Tweaks that are generally applied to all firesim configs setting a single clock domain at 1000 MHz
 class WithFireSimConfigTweaks extends Config(
+  new chipyard.config.WithUART(txEntries=128, rxEntries=128) ++ // FireSim requires a larger UART FIFO buffer, 
```

Comment:
Keeping the same functionality needs 256 entries on both. Also, this should be moved into the `WithFireSimDesignTweaks` (this frag. is mainly used for frequency changes).

Reply:
fixed